apps/web/lang/en.json

```
@@ -192,6 +192,16 @@
         "affiliateProgram": "Affiliate program",
         "suppliers": "Suppliers"
       }
+    },
+    "legal": {
+      "label": "Legal",
+      "subcategories": {
+        "termsAndConditions": "Terms and Conditions",
+        "cancellationRights": "Cancellation Rights",
+        "legalDisclosure": "Legal Disclosure",
+        "privacyPolicy": "Privacy Policy",
+        "cancellationForm": "Cancellation Form"
+      }
```

```suggestion
    "legal": {
      "label": "Legal",
      "subcategories": {
        "cancellationForm": "Cancellation form",
        "cancellationRights": "Cancellation rights",
        "legalDisclosure": "Legal disclosure",
        "privacyPolicy": "Privacy policy",
        "termsAndConditions": "Terms and conditions",
      }
```
**todo**: Also add translations to `de.json`.
```
    "legal": {
      "label": "Rechtliches",
      "subcategories": {
        "cancellationForm": "Widerrufsformular",
        "cancellationRights": "Widerrufsbelehrung",
        "legalDisclosure": "Impressum",
        "privacyPolicy": "Datenschutzerklärung",
        "termsAndConditions": "AGB"
      }
```